plugins/GSdx/GLLoader.cpp

```diff
@@ -367,7 +367,12 @@ namespace GLLoader {
 #ifdef _WIN32
 		if (status) {
 			if (fglrx_buggy_driver) {
-				fprintf(stderr, "OpenGL renderer is slow on AMD GPU due to inefficient driver. Sorry.\n");
+				fprintf(stderr, "OpenGL renderer is slow on AMD GPU due to inefficient driver.Please check the links below.\n"
```

Comment:
I would prefer if ``Please check the links below`` is changed to ``Please check out the links listed below``. Also, ``OpenGL renderer`` **->** ``OpenGL hardware renderer``.